### ISSUE TITLE
Fix is_player_count_max for playoffs

### ISSUE THREAD

**Issue by wallace123** [wallace123/fnfldawgs#6]
Current solution just turns off the max check count once the playoffs start and places the responsibility on the user. Need a better solution in fnfl_helpers.py function is_player_count_max.

A player cannot be used more than four times in the season except for in the playoffs. However, it's not that simple. Easier to explain by example.

Example 1: Cam Newton has been played 4 times in the regular season (Week 1 - Week 17). If a user tries to add him to a playoff lineup (Wild Card - Super Bowl), this should fail.

Example 2: Cam Newton has been played 3 times in the regular season (Week 1 - Week 17). If a user tries to add him to a playoff lineup (Wild Card - Super Bowl), this should succeed. Playoff lineups continue throughout the playoffs with the exception of three changes. So Cam Newton could be played throughout the playoffs which means his usage count will exceed the max (4 times). This should also succeed.

So basically we need a player count for the regular season. If a player has not reached the max by playoff time, he can be added to a playoff lineup.

**Comment by wallace123** [comment]
Updated helper functions _get_all_players and get_player_count to be regular season aware. Now is_player_count_max only gets a count of players from the regular season so players can be used more than the max in the playoffs if they started under the max at playoff start time.
